branchinfo.txt

```diff
@@ -3,7 +3,7 @@
 # Each line is expected to be in the format "[Name]=[Value]".
 MAJOR_VERSION=1
 MINOR_VERSION=0
-PATCH_VERSION=3
+PATCH_VERSION=4
 RELEASE_SUFFIX=rc4
```

Comment:
Yep, that's the next one. 1.0.3 is the latest out there. What are your concerns? I may be missing something.